frontend/integration-tests/tests/kubevirt/utils/consts.ts

```diff
@@ -1,6 +1,16 @@
 export const DASHES = '---';
 export const STORAGE_CLASS = process.env.STORAGE_CLASS;
 
+export const KUBEADMIN_IDP = 'kube:admin';
+export const KUBEADMIN_USERNAME = 'kubeadmin';
+export const {
+  BRIDGE_HTPASSWD_IDP = 'test',
+  BRIDGE_HTPASSWD_USERNAME = 'test',
+  BRIDGE_HTPASSWD_PASSWORD = 'test',
```

Comment:
can the values at least be different? so they don't get switched by accident

Reply:
It can be different, but I don't see the necessary to do it.
Openshift QE deployment job are using these values as default, with these values it can work with openshift qe env as well.

Reply:
ok